ci-operator/jobs/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-postsubmits.yaml

```diff
@@ -30,33 +30,3 @@ postsubmits:
           requests:
             cpu: 10m
       serviceAccountName: ci-operator
-  - agent: kubernetes
```

Comment:
Please open a separate PR if you want to move the configuration from `openshift-kubernetes-autoscaler-postsubmits.yaml` to `openshift-kubernetes-autoscaler-postsubmits-release-3.10.yaml`.